maps/biter_battles_v2/feed_values_tab.lua

```diff
@@ -0,0 +1,170 @@
+-- feed values tab --
+
+local Tabs = require 'comfy_panel.main'
+local tables = require "maps.biter_battles_v2.tables"
+local event = require 'utils.event'
+local bb_config = require "maps.biter_battles_v2.config"
+local food_values = tables.food_values
+local food_long_and_short = tables.food_long_and_short
+local food_long_to_short = tables.food_long_to_short
+local forces_list = tables.forces_list
+local science_list = tables.science_list
+local evofilter_list = tables.evofilter_list
+local food_value_table_version = tables.food_value_table_version
```

Comment:
```suggestion
local tables = require "maps.biter_battles_v2.tables"
local food_values = tables.food_values
local food_long_and_short = tables.food_long_and_short
```

lots of this is unused

Reply:
Done